application/views/admin/survey/subview/accordion/_tokens_panel.php

```diff
@@ -45,6 +45,61 @@ function alertPrivacy()
     
 ",
     LSYii_ClientScript::POS_BEGIN);
+
+App()->getClientScript()->registerScript("edit-after-completion-message", "
```

Comment:
No way to add the sentence in HTML directly inside block and just shown/hide good blick (div or p) according to the current settings.

More easy to move to extrenal JS after.
Stgart by show a string for both and dynamicall hide/show 

```
            <div class="mb-3">
                <label class=" form-label" for='alloweditaftercompletion' title='<?php  eT("If participant-based response persistence is enabled a participant can update his response after completion, otherwise a participant can add new responses without restriction."); ?>'>
                    <?php  eT("Allow multiple responses or update responses with one access code:"); ?>
                </label>
                <div>
                    <?php $this->widget('ext.ButtonGroupWidget.ButtonGroupWidget', [
                        'name'          => 'alloweditaftercompletion',
                        'checkedOption' => $oSurvey->alloweditaftercompletion,
                        'selectOptions' => ($bShowInherited)
                            ? array_merge($optionsOnOff, ['I' => $oSurveyOptions->alloweditaftercompletion . " ᴵ"])
                            : $optionsOnOff
                    ]); ?>
                </div>
                <div id="alloweditaftercompletion-base" class="form-text"><?php eT('Allow multiple responses or update the responses using the access code'); ?></div>
                <div id="alloweditaftercompletion-update" class="form-text hidden"><?php eT('Allow to update the responses using the access code'); ?></div>
                <div id="alloweditaftercompletion-multiple" class="form-text hidden"><?php eT('Allow multiple responses with the same access code'); ?></div>
            </div>
```

Reply:
> No way to add the sentence in HTML directly inside block and just shown/hide good blick (div or p) according to the current settings ?

If that's the case, you can just move the block code to extenral JS and leave the language definition here.

Reply:
? In my proposed solution : JS do JS , no language …

In your proposed solution you create var for language string… you can not leave the language definition here …